test/test_datasets.py

```diff
@@ -2665,5 +2665,27 @@ def inject_fake_data(self, tmpdir: str, config):
         return num_images
 
 
+class RenderedSST2TestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.RenderedSST2
+    FEATURE_TYPES = (PIL.Image.Image, int)
```

Comment:
Nit: we don't need this line as it's the default of the `datasets_utils.ImageDatasetTestCase` class